ui.tests/test-module/.eslintrc.js

```diff
@@ -13,14 +10,11 @@ module.exports = {
         ecmaVersion: 9
     },
     rules: {
-        'semi': ['error'],
+        semi: ['error'],
```

Comment:
Why are single quotes removed here for some of the rule entries? Should be either all or non? How is that related to your change?

Reply:
That's also the result of the prettier reformatting. I suppose the quotes are needed for the properties with '-' in their name.

Reply:
As I wrote in the PR description the relevant new JS file containing the tests is content-fragment-component-dialog.js .